app/models.py

```diff
@@ -986,7 +991,7 @@ def _as_utils_template_with_personalisation(self, values):
         template.values = values
         return template
 
-    def serialize(self):
+    def serialize_for_v2(self):
```

Comment:
Why change the name? For clarity?

Reply:
yup - i did it to validate to myself that no-where else uses `serialize` (that might need broadcast stuff for admin)